packages/yargs/mixins/log/mixin.core.js

```diff
@@ -30,36 +38,22 @@ module.exports = class CLIMixin extends Mixin {
   handleArguments(argv) {
     this.options = { ...this.options, ...argv };
     const { quiet, verbose } = this.options;
-    this.logger.setLogLevel(logLevels.info + verbose - quiet);
-    this.logger.info(
-      `started in ${process.env.NODE_ENV || 'development'} mode`
-    );
+    const logger = this.getLogger();
+    logger.setLogLevel(logLevels.info + verbose - quiet);
+    logger.info(`started in ${process.env.NODE_ENV || 'development'} mode`);
```

Comment:
This has been irritating me for quite some time - it always says `started in xxx mode`, but maybe it would be nice to change the wording based on which command was executed (`build`, `start`, `serve`).